### ISSUE TITLE
host on netlify for auto-build

### ISSUE THREAD

**Issue by dckc** [dckc/madmode-blog#54]
A lot of the friction around my blog is running the build step: re-constituting python dependencies, running the build, manually tweaking the results, and committing the build results to github pages.

I really enjoyed Netlify's build-on-push workflow when working on the library of RChain.

Let's give it a try...

**Comment by dckc** [comment]
holy shnikeys! it worked 1st try with no tweaks whatsoever

https://madmode-blog-dckc.netlify.com/

[![Netlify Status](https://api.netlify.com/api/v1/badges/76b84411-2064-48f7-afaa-aed98e369c38/deploy-status)](https://app.netlify.com/sites/madmode-blog-dckc/deploys)

https://app.netlify.com/sites/madmode-blog-dckc/deploys/5ce76401042cfd6c4a47eab8

lots of mixed content (http / https) issues:
```
10:26:09 PM: Mixed content detected in: /2000_09_01_archive.html
10:26:09 PM: --> insecure script urls:
10:26:09 PM:   - http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML

10:27:16 PM: Mixed content detected in: /2010/09/ditch-cable-tv-yes-build-hd-dvr-out-of.html
10:27:16 PM: --> insecure img urls:
10:27:16 PM:   - http://www.assoc-amazon.com/e/ir?t=danconnolly&amp;l=btl&amp;camp=213689&amp;creative=392969&amp;o=1&amp;a=B001GGAIIQ
```

**Comment by dckc** [comment]
fixed, documented in 7b4e659
